Mathlib/Topology/LocallyConstant/Algebra.lean

```diff
@@ -291,48 +291,133 @@ theorem coe_algebraMap (r : R) : ⇑(algebraMap R (LocallyConstant X Y) r) = alg
 
 end Algebra
 
+section coeFn
```

Comment:
Could you move `coeFnMonoidHom` to this section too?

Reply:
This means the `DistribMulAction` and `Module` instances have to move to this section as well (they are necessary to define `coeFnₗ` and depend on `coeFnAddMonoidHom`). I'm fine with that change and I'll push it, but I don't feel strongly about it and can move them back if needed.

Reply:
Ah it's probably best not to move anything then, sorry!